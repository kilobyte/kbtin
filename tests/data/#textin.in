#sys {rm -rf testtmp}
#ali x #deathlog testtmp
x abc
x
x def
x
x ghi
x
x
x jkl
x
x {#abc;#bde}
x ąęść
x abc𠀧𠀀𠀁𠀂
x
#delay 1 #end
#run a //selfpipe
#text testtmp
#unlink testtmp
