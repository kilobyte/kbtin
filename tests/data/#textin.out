#File read - Success.
abc

def

ghi


jkl

#abc;#bde
ąęść
abc𠀧𠀀𠀁𠀂


#SESSION 'a' DIED.
